code/modules/admin/admin_verbs.dm

```diff
@@ -79,6 +79,7 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	/client/proc/stabilize_atmos,
 	/client/proc/openTicketManager,
 	/client/proc/fix_air,
+	/client/proc/kill_rads,
```

Comment:
austation note needed

Reply:
terra doesn't have it on austation verbs he added

Reply:
he should have xd

Reply:
🤷

Reply:
nerds, read the next couple of lines, there is a fucking comment
it's not in the list because it fucks with the newlines and causes errors

Reply:
Yeah I see it. This still needs a comment though, it doesn't have to be on the same line